components/tyk-enterprise-portal/templates/statefulset-enterprise-portal.yaml

```diff
@@ -0,0 +1,235 @@
+          privileged: false
+          readOnlyRootFilesystem: true
+          capabilities:
+            drop:
+              - all
+        env:
+          - name: PORTAL_LICENSEKEY
+            valueFrom:
+              secretKeyRef:
+                {{ if .Values.secrets.useSecretName }}
+                name: {{ .Values.secrets.useSecretName }}
+                {{ else }}
+                name: secrets-{{ include "tyk-enterprise-portal.fullname" . }}
+                {{ end }}
+
+                key: EnterprisePortalLicense
+          - name: PORTAL_HOST_PORT
+            value: "{{ .Values.containerPort }}"
+          - name: ADMIN_EMAIL
+            value: "{{ .Values.global.adminUser.email }}"
+          - name: ADMIN_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                {{ if .Values.secrets.useSecretName }}
+                name: {{ .Values.secrets.useSecretName }}
+                {{ else }}
+                name: secrets-{{ include "tyk-enterprise-portal.fullname" . }}
+                {{ end }}
+                key: EnterprisePortalAdminPassword
+          - name: PORTAL_DATABASE_DIALECT
+            value: "{{ .Values.storage.database.type }}"
+          - name: PORTAL_DATABASE_CONNECTIONSTRING
+            valueFrom:
+              secretKeyRef:
+                {{ if .Values.secrets.useSecretName }}
+                name: {{ .Values.secrets.useSecretName }}
+                {{ else }}
+                name: secrets-{{ include "tyk-enterprise-portal.fullname" . }}
+                {{ end }}
+                key: EnterprisePortalStorgeConnectionString
+          - name: PORTAL_THEMING_THEME
+            value: "default"
+          - name: PORTAL_THEMING_PATH
+            value: "./themes"
+          - name: PROVIDER_NAME
+            value: "tyk"
+          - name: TYK_URL
+            value: "http{{ if .Values.global.tls.enterprisePortal }}s{{ end }}://dashboard-svc-{{ .Release.Name }}.{{ .Release.Namespace }}.svc:{{ .Values.global.servicePorts.dashboard }}"
+          - name: TYK_AUTH
+            valueFrom:
+              secretKeyRef:
+                {{ if .Values.secrets.useSecretName }}
+                name: {{ .Values.secrets.useSecretName }}
+                {{ else if .Values.global.bootstrap.enterprisePortal}}
+                name: {{ .Values.global.secrets.enterprisePortal}}
+                {{ else }}
+                name: secrets-{{ include "tyk-enterprise-portal.fullname" . }}
+                {{ end }}
+                key: TYK_AUTH
+          - name: TYK_ORG
+            valueFrom:
+              secretKeyRef:
+                {{ if .Values.secrets.useSecretName }}
+                name: {{ .Values.secrets.useSecretName }}
+                {{ else if .Values.global.bootstrap.enterprisePortal}}
+                name: {{ .Values.global.secrets.enterprisePortal}}
+                {{ else }}
+                name: secrets-{{ include "tyk-enterprise-portal.fullname" . }}
```

Comment:
`TYK_ORG` is not set in the secret of enterprise portal.

Reply:
Fixed